modules/virt-enabling-load-balancer-service-web.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
```

Comment:
@phoracek New module needs to be reviewed. I broke it out of the "Enabling services in UI" procedure because this procedure requires cluster-admin permissions while the other procedure does not.